bdsx/bds/packets.ts

```diff
@@ -664,7 +671,12 @@ export namespace AnimatePacket {
 
 @nativeClass(null)
 export class RespawnPacket extends Packet {
-    // unknown
+    @nativeField(Vec3)
+    position:Vec3;
```

Comment:
Better to be `pos` for consistency? btw IDA shows `mPos`, `mState`, `mRuntimeId`

Reply:
I can't find these names myself on IDA, but I will change them